.github/workflows/watch-conda-forge.yml

```diff
@@ -0,0 +1,43 @@
+name: Check for new versions of pangeo on Conda Forge
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    if: github.repository == 'pangeo-gallery/default-binder'
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get latest pangeo version
+        id: latest_version
+        uses: jacobtomlinson/gha-anaconda-package-version@0.1.1
+        with:
+          org: "conda-forge"
+          package: "pangeo-notebook"
+      - name: Find and Replace Base
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "binder/"
+          find: "FROM pangeo/base-image: "
+          replace: "FROM pangeo/base-image:${{ steps.latest_version.outputs.version }} "
+      - name: Find and Replace Notebook
+        uses: jacobtomlinson/gha-find-replace@0.1.1
+        with:
+          include: "binder/"
+          find: "pangeo-notebook==.* "
+          replace: "pangeo-notebook==${{ steps.latest_version.outputs.version }} "
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "Update pangeo-notebook version to ${{ steps.latest_version.outputs.version }}"
+          title: "Update pangeo-notebook to ${{ steps.latest_version.outputs.version }}"
+          reviewers: "TomAugspurger"
+          branch: "upgrade-pangeo-version"
+          body: |
+            A new pangeo version has been detected.
+
+            Updated environment to use `${{ steps.latest_version.outputs.version }}`.
```

Comment:
Rather than a PR, I would rather see a direct commit with a tag.

Reply:
Agreed that we want to automatically create a tag on commits / PRs made by this bot. I'm somewhat hesitant to have bots pushing straight to master (robots taking my job!) so I'd like to do that in a separate workflow if you don't mind.

Looking into repository dispatch now.

Reply:
@rabernat for repository dispatch, would I install that action on the repository *sending* the event (conda-forge/pangeo-notebook-feedstock in this case)? And the webhook is on the receiving repository?

Reply:
correct. However, I think the sending repository is pangeo-stacks-dev, since we are not using the metapackage here any more.